File: C/266A-Stones on the Table.c
/*
    Solution to Codeforces problem 266A
    Copyright (c) GeneralMing. All rights reserved.

    https://github.com/GeneralMing/codeforces
*/

#include<stdio.h>
#include<string.h>
 
int main()
{
    int n, count = 0;
    
    scanf("%d", &n);
    char str[n];
    scanf("%s", str);
    
    for(int i= 0; i < n-1 ; i++)
        if(str[i] == str[i+1])
            count++;
    
    printf("%d\n", count);
    return 0;
}

File: Java/266A-Stones on the Table.java
/*
    Solution to Codeforces problem 266A
    Copyright (c) GeneralMing. All rights reserved.

    https://github.com/GeneralMing/codeforces
*/

import java.util.*;
public class Stone_on_the_Table
{
	public static void main(String[] args)
	{
		int n, count = 0;
		Scanner in = new Scanner(System.in);
		n = Integer.parseInt(in.nextLine());
		String str;
		str = in.nextLine();
		for(int i = 0; i < n-1; i++)
			if(str.charAt(i) == str.charAt(i+1))
				count++;
		System.out.println(count);
	}
}